src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractServiceImpl.java

```diff
@@ -168,7 +171,11 @@ private PdpFeedFileBaseEntry buildPdpFeedFileBaseEntry(ConcurStandardAccountingE
                     totalReimbursementDollarAmount = totalReimbursementDollarAmount.add(line.getJournalAmount());
                 }
                 logJournalAccountCodeOverridden(line, reportData);
-                if (getConcurStandardAccountingExtractValidationService().validateConcurStandardAccountingExtractDetailLine(line, reportData)) {
+                String overriddenObjectCode = getParameterService().getParameterValueAsString(CUKFSConstants.ParameterNamespaces.CONCUR, 
+                        CUKFSParameterKeyConstants.ALL_COMPONENTS, ConcurParameterConstants.CONCUR_SAE_PDP_DEFAULT_OBJECT_CODE);
+                String overriddenSubObjectCode = StringUtils.EMPTY;
+                if (getConcurStandardAccountingExtractValidationService().validateConcurStandardAccountingExtractDetailLineWithObjectCodeOverride(line, reportData, 
+                        overriddenObjectCode, overriddenObjectCode)) {
```

Comment:
Based on the new method signature, I think the last argument should be the sub-object code, not the object code.